packages/ui-responsive/src/Responsive/index.js

```diff
@@ -193,7 +206,7 @@ class Responsive extends Component {
       renderFunc = children || render
     }
     return (
-      <div>
+      <div style={{ display }}>
```

Comment:
Not very elegant, but I decided to go with inline style, because it is not worth making the component themeable for this, especially in V7.